.travis.yml

```diff
@@ -1,17 +1,17 @@
-sudo: false
+sudo: required
 language: php
 php:
   - 5.5
   - 5.6
 
 # Set environment variables
 node_js:
-  - "0.12"
+  - "4.1"
 services:
   - mysql
 env:
   matrix:
-  - APP_ENV=testing DB_HOST=localhost DB_DATABASE=madison_testing DB_USERNAME=root
+  - APP_URL=http://0.0.0.0:8888 APP_ENV=testing DB_HOST=localhost DB_DATABASE=madison_testing DB_USERNAME=root APP_DEBUG=true
```

Comment:
Does it make sense to use debug here?  I'm worried we're not testing the production-build assets.